assets/js/app/Instance.js

```diff
@@ -18,46 +18,57 @@ LocusZoom.Instance = function(id, datasource, layout, state) {
     this.initialized = false;
 
     this.id = id;
-    this.parent = LocusZoom;
     
     this.svg = null;
 
-    // The _panels property stores child panel instances
-    this._panels = {};
+    // The panels property stores child panel instances
+    this.panels = {};
     this.remap_promises = [];
+
+    // The layout is a serializable object used to describe the composition of the instance
+    this.layout = layout || JSON.parse(JSON.stringify(LocusZoom.DefaultLayout));
```

Comment:
Good call... as a corollary to this I'd like to make the layout itself more of a "living" object such that you could change values and reload the layout on the fly. Right now you can do this only by destroying and recreating the instance.

I've been kicking around the idea of building a Layout class. It would need methods to spit out or consume a JSON object to keep the portability we've introduced in this branch, but would also potentially help address this concern of defining/consuming partial layouts or changing layout parameters on the fly by keeping all the associated methods in one place. That's a pretty big iteration on the more basic layout approach taken in this branch, though, so should probably be done in a subsequent branch.